packages/skills-gameon-sdk/src/utils/displayUtils.ts

```diff
@@ -52,6 +52,15 @@ interface RenderOptions {
      */
     secondaryColor?: string;
 
+    /**
+     * Leaderboard score primary text, default to `You placed ${player.score.ordinalRank}!` if not supplied
+     */
+    scorePrimaryText?: string;
+
+    /**
+     * Leaderboard score secondary text, default to `${player.score.score} points` if not supplied
+     */
+    scoreSecondaryText?: string;
```

Comment:
Nit: Can you make these comments match the style of the other properties? i.e. "Leaderboard score primary text. Will default to `You placed ${player.score.ordinalRank}!` if not supplied"